server/controllers/match/placementRequests/occupancyViewController.ts

```diff
@@ -29,32 +43,68 @@ export default class {
 
   view(): TypedRequestHandler<Request> {
     return async (req: NewRequest, res: Response) => {
-      const { startDate, durationDays, premisesName, premisesId, apType } = req.query
-      const dates = placementDates(startDate, durationDays)
-      const [placementRequest, capacity] = await Promise.all([
-        this.placementRequestService.getPlacementRequest(req.user.token, req.params.id),
-        this.premisesService.getCapacity(req.user.token, premisesId, dates.startDate, dates.endDate),
-      ])
-      const essentialCharacteristics = filterOutAPTypes(placementRequest.essentialCriteria)
-      const matchingDetailsSummaryList = occupancyViewSummaryListForMatchingDetails(
-        capacity.premise.bedCount,
-        dates,
-        placementRequest,
-        essentialCharacteristics,
-      )
-      const occupancySummaryHtml = occupancySummary(capacity)
-      const calendar = occupancyCalendar(capacity)
+      const { token } = req.user
+      const { id, premisesId } = req.params
+      const { apType, criteria, ...filterUserInput } = req.query
```

Comment:
So `apType` only seems to be used to populate the hidden fields. I'm thinking it could be refactored out - but not in this PR